scripts/src/modules/search/mobile-filter-expander.js

```diff
@@ -14,8 +15,12 @@ export default function() {
     let $showHideButton = document.createElement('button');
 
     //number of filters selected
-    let $noOfFilters = document.location.search.match(/=/g).length-8;
-    let $buttonHtml= "Filters <span class='filter-indicator'>"+ $noOfFilters +"</span>";
+    //let $noOfFilters = document.location.search.match(/=/g).length-8;
+
+
+
+
+
```

Comment:
Unused code, blank line can be removed. Sorry to bring this up on round2 of responses.